arbeitszeit_web/presenters/registration_email_presenter.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from typing import Protocol
+from uuid import UUID
+
+from arbeitszeit_web.email import EmailConfiguration, MailService, UserAddressBook
+from arbeitszeit_web.translator import Translator
+from arbeitszeit_web.url_index import ConfirmationUrlIndex
+
+
+class RegistrationEmailTemplate(Protocol):
+    def render_to_html(self, confirmation_url: str) -> str:
+        ...
+
+
+@dataclass
+class RegistrationEmailPresenter:
+    email_sender: MailService
+    address_book: UserAddressBook
+    member_email_template: RegistrationEmailTemplate
+    company_email_template: RegistrationEmailTemplate
```

Comment:
`company_email_template` is not used - in case you overlooked it. But probably we could leave it as is because later we surely will have different templates for companies and members.

Reply:
Thanks for catching this.